integreat_cms/static/src/fonts/noto-sans-ethiopic/noto-sans-ethiopic.css

```diff
@@ -0,0 +1,35 @@
+/*
+ * Noto Sans Ethiopic
+ *
+ * This font is used for chinese in the content (tinymce, preview and PDF).
+ *
+ * Generated by GoogleWebfontsHelper https://github.com/majodev/google-webfonts-helper
+ */
+
+/* noto-sans-sc-regular - ethiopic */
+@font-face {
+    font-family: "Noto Sans Ethiopic";
+    font-style: normal;
+    font-weight: normal;
+    src: url("./noto-sans-ethiopic-v42-latin-ext_latin_ethiopic-regular.eot"); /* IE9 Compat Modes */
+    src: local("Noto Sans Ethiopic"), local("NotoSansEthiopic-Regular"),
+        /* Local installed font */ url("./noto-sans-ethiopic-v42-latin-ext_latin_ethiopic-regular.eot?#iefix")
+            format("embedded-opentype"),
+        /* IE6-IE8 */ url("./noto-sans-ethiopic-v42-latin-ext_latin_ethiopic-regular.woff2") format("woff2"),
+        /* Super Modern Browsers */ url("./noto-sans-ethiopic-v42-latin-ext_latin_ethiopic-regular.woff") format("woff"),
+        /* Modern Browsers */ url("./noto-sans-ethiopic-v42-latin-ext_latin_ethiopic-regular.ttf") format("truetype"); /* Safari, Android, iOS */
+}
+
+/* noto-sans-sc-700 - chinese-simplified */
```

Comment:
I think this is just a copy-paste-issue. I think it's supposed to be "ethiopic -  simplified" or even "amharic-simplified", since their are a few languages spoken in Ethiopia which we might integrate in our system

Reply:
Oh 😓 Yes, this was looked over. I'll change it to "ethiopic".
(I've chosen the term "ethiopic" since the font set is called so. If you think "it's better to use the term "amharic", I'll change them.)